source/MRMesh/MRMesh.vcxproj

```diff
@@ -380,7 +380,6 @@
     <ClInclude Include="MRMeta.h" />
     <ClInclude Include="MRMacros.h" />
     <ClInclude Include="MRCanonicalTypedefs.h" />
-    <ClInclude Include="MRBindingMacros.h" />
```

Comment:
why?

Reply:
Moved to MRPch because I need it in `MRPch/MRFmt.h`.